docs/design_docs.md

```diff
@@ -0,0 +1,22 @@
+# Jujutsu Design Docs
+
+Jujutsu uses Design Docs to drive technical decisions on large projects and it
+is the place to discuss your proposed design or new component. It is a very 
+thorough process, in which all concerns must be addressed. It shares some
+similarities with [Rust RFCs] but mostly addresses _technical_ problems and 
+gauges the technical and social concerns of all stakeholders.
+
+So if you want to start building the native backend or the server component for
+Jujutsu, you'll need to go through this process. It also is good if you're 
+already familiar with the codebase, to not stall the implementation.
```

Comment:
Maybe something like "It's also good if you're already familiar with the codebase" => "It's good to go through the process even if you're already familiar with the codebase". I wasn't sure if you meant that or if you meant that one should get familiar with the codebase first ("It's good if you're familiar with the codebase before starting work").

Reply:
I intended to say the second thing, as people unfamiliar with the codebase shouldn't propose large changes.

Reply:
now rewritten as that

Reply:
I see. I don't see how that's very related to the previous sentence in the paragraph. And isn't it true whether or not they're writing a design doc? That is, it seems like it belongs more in `contributing.md` than here, but it also seems obvious enough that I don't think we need to even mention it.

Reply:
Ok, should I remove or rewrite it to your first proposed variant?

Reply:
Just remove it IMO, but feel free to say you disagree, of course

Reply:
removed